docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,498 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    desired0_idx, desired1_idx = 550, 2030
</pre><p>Please move this line to the top of the next code cell, where you show the desired instances. Also, please add one blank line after it for sake of readability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>